bin/raptor-regression

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+var raptorRegression = require('..');
+var userArgs = process.argv;
+
+if (userArgs.length < 4) {
+  console.log("USAGE: %s data.json app.context <app.entrypoint>", userArgs[0]);
+  process.exit(1);
+}
+
+var dataFile = userArgs[2];
+var appContext = userArgs[3];
+if (userArgs.length == 5) {
+  var appEntrypoint = userArgs[4];  
+} else {
+  var appEntrypoint = 'none';
+}
+
+raptorRegression(dataFile, appContext, appEntrypoint);
```

Comment:
A catch here doesn't catch an error that is thrown in index.js:raptorRegression ie. your comment above. So just putting the catch in index.js